### PR TITLE
Update gherking file to include latest changes in the performance command

### PR DESCRIPTION
**Pull request by jredrejo**
### Summary
Gherking story for the new performance command was added in #4434 
Recently, there have been some changes in the behaviour of this command.
This PR includes these changes in the Gherking story

### Reviewer guidance
Open integration_testing/features/super-admin/super-admin-profile-command.feature
and read the story checking its syntax and logic.

### References
Relates #4434 

### Contributor Checklist

- [x] Contributor has fully tested the PR manually
- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

File: integration_testing/features/super-admin/super-admin-profile-command.feature [hunk 1]
```diff
@@ -8,13 +8,15 @@ Feature: Super Admin profile manage command
   Scenario: A terminal is open in the same machine where Kolibri is running
```
Comment (radinamatic) [inline integration_testing/features/super-admin/super-admin-profile-command.feature:8]: "Scenario name" should be more specific, even in cases when there is only one per `feature` file. How about something along the lines of `Execute the profile command and review the created CSV files`? 

Or separate actions in 2 or more scenarios: one to execute the command and check if it actually created the `_performance.csv`  file; second one to check if the file `_requests_performance.csv` file was created after some interactions; third one to open the files in text/spreadsheet editor and check if the recorded information is correct.

For a good example of small(er) steps described in separate scenarios in one feature file, check this:
https://github.com/learningequality/kolibri/blob/develop/integration_testing/features/coach/coach-lesson-resources-search.feature

File: integration_testing/features/super-admin/super-admin-profile-command.feature [hunk 2]
```diff
@@ -8,13 +8,15 @@ Feature: Super Admin profile manage command
...
     Given that I have access to the kolibri command line
```
Comment (radinamatic) [inline integration_testing/features/super-admin/super-admin-profile-command.feature:9]: This "given" sounds a bit unnecessary, considering there is one on line 6.

**Review by radinamatic — CHANGES_REQUESTED** [review]
Overly looks good, but I would recommend separating this sequence in 3 shorter scenarios for clarity.

**Comment by indirectlylit** [comment]
I'm going to remove the milestone here, but @jredrejo would you mind following up on @radinamatic's feedback?

**Comment by radinamatic** [comment]
Yes, this should not be a release blocker!
@jredrejo Let me know if the comments are clear :slightly_smiling_face:

**Comment by radinamatic** [comment]
Apologies, it was easier for me to make a rewrite here:

```
Feature: Super Admin runs manage profile command
    Super Admin needs to be able to generate and review profile logs on the device running Kolibri

  Background:
    Given that the Kolibri server is running
      And the '[Server]' section of the Kolibri's "options.ini" file is configured with 'PROFILE = 1' to enable request profiling

  Scenario: Execute the profile command and review the created "_performance.csv" file
    When I run the 'kolibri manage profile --num_samples=6' command in the Terminal
      And I browse the "KOLIBRI_HOME/performance folder"      # Usually $HOME/.kolibri/performance
    Then I see a "xxxxxxxx_xxxxxx_performance.csv" file       # xxxxxxxx_xxxxxx is current date_time, for example 20181022_194415_performance.csv

  Scenario: Add more interactions and review the created "_requests_performance.csv" file 
    When I interact with Kolibri in the browser by navigating through other server pages for at least 1 minute
      And I browse the "KOLIBRI_HOME/performance" folder again
    Then I see another file called "xxxxxxxx_xxxxxx_requests_performance.csv"
      And it has the same date_time as the "xxxxxxxx_xxxxxx_performance.csv" file

  Scenario: Stop interacting with server and observe profiling results 
    Given that 1 minute has passed after running the command 
    When I see the prompt in the Terminal again # Profiling requests is finished, and CSV files stop changing
    Then I open CSV files in a text editor or a spreadsheet application
      And I see that "xxxxxxxx_xxxxxx_performance.csv" file has at least 6 lines beginning with a timestamp followed by profiling results # First line is a header
      And I see that "xxxxxxxx_xxxxxx_requests_performance.csv" file has at least one line per each Kolibri page I interacted with in the browser, and lines begin with a timestamp followed by requests profiling results
```

I should add some conventions for Terminal commands, file and folder names to the `.feature` files style guide in the `README.md`.

**Comment by indirectlylit** [comment]
superseded by https://github.com/learningequality/kolibri/pull/4693/files#diff-988ae6c2c023ddfe66553564bf4a8f6dR8

**Comment by radinamatic** [comment]
Yes, this has been fixed by the latest update of `.feature` files #4693.

**Pull request closed without merging**